js/components/Auth.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @format
+ * @jsxImportSource theme-ui
```

Comment:
I'd like to avoid comments that change how code runs. It's also hard to google for this if you don't already know what jsx pragmas are in nextjs.

Reply:
Would something like this work just as well? https://theme-ui.com/guides/jsx-pragma#using-nextjs